php/libraries/Candidate.class.inc

```diff
@@ -657,17 +657,52 @@ class Candidate
     }
 
     /**
-     * Returns the candidate's calculated age today
+     * Returns the candidate's calculated age today in years, months, days
      *
-     * @return int The candidate's age today
+     * @return array The candidate's age today in an array with values for keys 'year',
+     *               'mon', and 'day'
      */
-    public function getAge(): int
+    public function getAge(): array
     {
-        $ageArray = \Utility::calculateAge(
+        return \Utility::calculateAge(
             $this->candidateInfo["DoB"],
             (new DateTime())->format('Y-m-d')
         );
-        return $ageArray['year'];
+    }
+
+    /**
+     * Returns the candidate's calculated age today in int years
+     *
+     * @return int The candidate's age today as years
+     */
+    public function getAgeYears(): int
+    {
+        return $this->getAge()['year'];
+    }
+
+    /**
+     * Returns the candidate's calculated age today in total number of months
+     *
+     * @return int The candidate's age today in months
+     */
+    public function getAgeInMonths(): int
+    {
+        $age    = $this->getAge();
+        $months = $age['year']*12 + $age['mon'] + ($age['day']/30);
+
+        return (round($months*10) / 10.0);
+    }
+
+    /**
+     * Returns the candidate's calculated age today in total number of days
+     *
+     * @return int The candidate's age today in days
+     */
+    public function getAgeInDays(): int
```

Comment:
Maybe this function shouldn't exist because we can't actually give any precision based on days.

Reply:
@johnsaigle hold your horses, i have another commit coming, and then i'll update the PR description

Reply:
I still think this is true even with DateTime. `calculateAge()` makes all months equal to 30 days so it's misleading to say we can provide precision at the level of days.

Reply:
i don't understand your comment. we're not using calculateAge() anymore here. also, the the number of days that I get from using DateTime diff is different to when using the results from calculateAge() and averaging the number of days in a month to be 30.

how do you know that DateTime does it this way without precision?

Reply:
@johnsaigle this is the document for formatting by ->days https://www.php.net/manual/en/class.dateinterval.php#dateinterval.props.days. it looks pretty legit to me. i'll do more research on it

Reply:
Sorry this is leftover from before when getAge was using calculateAge. This is fine now.